### PR TITLE
Added error messages from Validator & Applied fixes from StyleCI

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "laravelcollective/html",
+  "name": "andrey-helldar/html",
```
Comment (arthurkirkosa) [inline composer.json]: why rename the package?

Comment (andrey-helldar) [inline composer.json]: OMG. I use this module in your project by connecting it through the composer. If you leave "laravelcollective / html", an error occurs. That renamed.
Before you pull request forgot to rename to the correct name.

File: composer.json [hunk 2]
```diff
@@ -9,6 +9,10 @@
     {
       "name": "Taylor Otwell",
       "email": "taylorotwell@gmail.com"
+    },
+    {
+      "name": "Andrey Helldar",
+      "email": "helldar@ai-rus.com"
```
Comment (tshafer) [inline composer.json]: Remove please.

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,4 @@ composer.lock
 .DS_Store
 Thumbs.db
 .idea
+nbproject
```
Comment (tshafer) [inline .gitignore]: Remove please.

Comment (andrey-helldar) [inline .gitignore]: Здравствуйте!

Ваше письмо принято в обработку. Вам постараются ответить в кратчайшие сроки.

С уважением
     автоответчик

**Comment by tshafer** [comment]
Please resubmit without all the style changes.

**Pull request closed without merging**